.github/workflows/push.yml

```diff
@@ -11,6 +11,7 @@ on:
 jobs:
   # for each PR merge, trigger image build
   image:
+    if: github.repository == 'sustainable-computing-io/kepler'
```

Comment:
if so.... 
https://github.com/sustainable-computing-io/kepler/pull/1327/files#diff-f3fc934cf0d89bdf07de358896ff90f1202585df812cf606206d1830a9949811R20
https://github.com/sustainable-computing-io/kepler/pull/1327/files#diff-f3fc934cf0d89bdf07de358896ff90f1202585df812cf606206d1830a9949811R25
https://github.com/sustainable-computing-io/kepler/pull/1327/files#diff-f3fc934cf0d89bdf07de358896ff90f1202585df812cf606206d1830a9949811R25
also need for this check.

Reply:
I can add same condition to them, but they succeeded in this PR at least. That may be because it does not include code changes, but I'm not sure.

So should I add?

Reply:
from the condition, which means when you merge a PR on main branch and meet the specific path then the job been run as https://github.com/sustainable-computing-io/kepler/pull/1327/files#diff-f3fc934cf0d89bdf07de358896ff90f1202585df812cf606206d1830a9949811R4-R9
so, in fact, the changes on this file .... considering with running at a fork repo. only been triggered when the fork owner merge a PR at the fork side.

Reply:
for example, as https://github.com/SamYuan1990/kepler/pull/8 this is a PR at fork side, and once the pr merged. this pipeline been triggered..... if we only focus on daily build, we should not change on this file.

Reply:
> for example, as [SamYuan1990#8](https://github.com/SamYuan1990/kepler/pull/8) this is a PR at fork side, and once the pr merged. this pipeline been triggered..... if we only focus on daily build, we should not change on this file.

You are right. I added this change because I just found it. So I can remove if it needs.

Reply:
IMO, I recommend PR back to upstream, but not a PR just on a specific fork.
do we have any specific case to merge a PR at local only?

Reply:
ok, so let's just have daily.yml in this PR. :-)

Reply:
I reverted change in push.yml.